### PR TITLE
feat: release-notes page

### PR DESCRIPTION
**Pull request by jfcurran**
WIP
https://github.com/ory-corp/marketing/issues/67

### CODE REVIEW

File: src/markdown/release-notes/2022-02.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+id: releasenotes-2022-02
+title: Release Notes 2022-02
+publishedAt: '2022-02-08'
+path: '/2022-02'
+description: 'foobar long text'
```
Comment (jfcurran) [inline src/markdown/release-notes/2022-02.md:6]: @RinorG add copy here

File: src/markdown/release-notes/2022-02.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+id: releasenotes-2022-02
+title: Release Notes 2022-02
+publishedAt: '2022-02-08'
```
Comment (jfcurran) [inline src/markdown/release-notes/2022-02.md:4]: @RinorG add copy here

File: src/page-content/content-release-notes.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+const ArrowRight = <i className={'ph-arrow-right-bold size16'} />
+
+export const seo: SeoProps = {
+  description: 'foo',
+  title: 'bar'
+}
+
+export const hero: HeroImageProps = {
+  id: 'release.hero',
+  title: <>Text release notes</>,
```
Comment (jfcurran) [inline src/page-content/content-release-notes.tsx:20]: @RinorG add copy here

File: src/page-content/content-release-notes.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  description: 'foobar',
+  buttons: [
```
Comment (jfcurran) [inline src/page-content/content-release-notes.tsx:22]: @RinorG make sure buttons point to correct destination

File: src/page-content/content-release-notes.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+export const featureList: ReleaselistProps = {
+  id: 'release.featurelist',
+  slant: true,
+  background: 'grey',
+  features: [
+    {
+      title: 'Latest releases',
+      description: (
+        <>
+          orbi faucibus tempus luctus. Sed cursus tortor quis facilisis
```
Comment (jfcurran) [inline src/page-content/content-release-notes.tsx:41]: @RinorG add copy here

File: src/page-content/content-release-notes.tsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  image: <img className="responsive" loading="lazy" alt="foo" src={Release} />
+}
+
+export const featureList: ReleaselistProps = {
+  id: 'release.featurelist',
+  slant: true,
+  background: 'grey',
+  features: [
+    {
+      title: 'Latest releases',
```
Comment (jfcurran) [inline src/page-content/content-release-notes.tsx:38]: @RinorG add copy here, might need something better

**Comment by BlueAcidFrog** [comment]
@RinorG 

Content
**Headline**: Get the latest release notes for new features and updates in Ory Cloud.
**Sub**: Ory Cloud release notes highlight all the recent product improvements we have worked on so you can easily stay up to date on what’s the latest.

**Review by zeropaper — APPROVED** [review]

**Pull request merged**